scripts/compose/grafana/provisioning/datasources/datasource.yaml

```diff
@@ -1,7 +1,13 @@
 apiVersion: 1
- 
+
```

Comment:
Commit message: Why? What's the rationale?

Reply:
As part of the rationale, maybe you could also *very* briefly mention what Graphite is?

Reply:
I have added a rationale which also gives a hint on what Graphite is.